common/flatpak-context.c

```diff
@@ -586,12 +586,13 @@ static gboolean
 get_xdg_user_dir_from_string (const char  *filesystem,
```

Comment:
Commit message: `Closes: #5920`

Reply:
what's wrong with the commit message?

Reply:
Everyone else uses `Closes: #5920`, you use `Close #5920`